electrum/gui/qt/paytoedit.py

```diff
@@ -127,10 +128,16 @@ def parse_script(self, x):
         return script
 
     def parse_amount(self, x):
-        if x.strip() == '!':
+        x_stripped = x.strip()
```

Comment:
let's just do `x = x.strip()`

Reply:
ok, I'll add changes in a half or hour

Reply:
`x_stripped` -> `x`, rebased fix commit with squash